crates/kornia-imgproc/src/flip.rs

```diff
@@ -27,43 +33,60 @@ use rayon::{iter::ParallelIterator, slice::ParallelSliceMut};
 /// )
 /// .unwrap();
 ///
-/// let flipped: Image<f32, 3> = horizontal_flip(&image).unwrap();
+/// let mut flipped = Image::<f32, 3>::from_size_val(
```

Comment:
```suggestion
/// let mut flipped = Image::<f32, 3>::from_size_val(image.size()
```
im adding this as a good practice to avoid shape issues